src/Cardano/Shell/NodeIPC/Example.hs

```diff
@@ -59,42 +59,52 @@ exampleWithFD = do
 
     (clientReadHandle, clientWriteHandle) <- getReadWriteHandles
 
-    (_, responses) <-
-        ipcServer clientWriteHandle Ping
+    (_, responses) <- do
+        (serverReadHandle, serverWriteHandle) <- getReadWriteHandles
+        -- Send message to server
+        sendMessage serverWriteHandle Ping
```

Comment:
You can parametrize the message so the user can choose whether to send `Ping`, `QueryPort` or something else we add.